atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1564,7 +1563,9 @@ private boolean rollbackOtherTransaction(long startTs, TransactionService transa
             return true;
         } catch (KeyAlreadyExistsException e) {
             String msg = "Two transactions tried to roll back someone else's request with start: " + startTs;
-            log.error("This isn't a bug but it should be very infrequent. " + msg,
+            log.error("This isn't a bug but it should be very infrequent. "
+                    + "Two transactions tried to roll back someone else's request with start: {}",
+                    startTs,
```

Comment:
This could just be `log.error("This isn't a bug but it should be very infrequent. {}", msg);`